src/_pytest/assertion/truncate.py

```diff
@@ -12,41 +12,54 @@
 
 
 DEFAULT_MAX_LINES = 8
-DEFAULT_MAX_CHARS = 8 * 80
+DEFAULT_MAX_CHARS = DEFAULT_MAX_LINES * 80
 USAGE_MSG = "use '-vv' to show"
 
 
-def truncate_if_required(
-    explanation: list[str], item: Item, max_length: int | None = None
-) -> list[str]:
+def truncate_if_required(explanation: list[str], item: Item) -> list[str]:
     """Truncate this assertion explanation if the given test item is eligible."""
-    if _should_truncate_item(item):
-        return _truncate_explanation(explanation)
+    should_truncate, max_lines, max_chars = _get_truncation_parameters(item)
+    if should_truncate:
+        return _truncate_explanation(
+            explanation,
+            max_lines=max_lines,
+            max_chars=max_chars,
+        )
     return explanation
 
 
-def _should_truncate_item(item: Item) -> bool:
+def _get_truncation_parameters(item: Item) -> tuple[bool, int, int]:
     """Whether or not this test item is eligible for truncation."""
```

Comment:
We should update the docstring:

```suggestion
    """Return the truncation parameters related to the given item, as (should truncate, max lines, max chars)."""
```